DataMigration/src/test/java/org/sagebionetoworks/dian/datamigration/SynapseTests.java

```diff
@@ -0,0 +1,307 @@
+ * may be used to endorse or promote products derived from this software without
+ * specific prior written permission. No license is granted to the trademarks of
+ * the copyright holders even if such marks are included in this software.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+ * AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+ * IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+ * ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR CONTRIBUTORS BE LIABLE
+ * FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+ * DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+ * SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+ * CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+ * OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+package org.sagebionetoworks.dian.datamigration;
+
+import com.google.common.collect.Lists;
+
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.sagebionetworks.client.SynapseClient;
+import org.sagebionetworks.client.SynapseClientImpl;
+import org.sagebionetworks.client.exceptions.SynapseException;
+import org.sagebionetworks.repo.model.EntityChildrenRequest;
+import org.sagebionetworks.repo.model.EntityChildrenResponse;
+import org.sagebionetworks.repo.model.EntityHeader;
+import org.sagebionetworks.repo.model.EntityType;
+import org.sagebionetworks.repo.model.FileEntity;
+import org.sagebionetworks.repo.model.Project;
+import org.sagebionetworks.repo.model.file.FileHandleAssociateType;
+import org.sagebionetworks.repo.model.file.FileHandleAssociation;
+import org.sagebionetworks.utils.MD5ChecksumHelper;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.Assert.*;
+
+public class SynapseTests {
+
+    private static SynapseClient synapse;
+
+    private static Project project;
+    private static String projectId = "syn25791302";
+
+    @BeforeClass
+    public static void beforeClass() throws Exception {
+        // Create 2 users
+        synapse = new SynapseClientImpl();
+        synapse.setBearerAuthorizationToken(SynapseUtil.synapsePersonalAccessToken);
+
+        project = new Project();
+        project.setId(projectId);
+    }
+
+    @Test
+    public void testNavigateFolderStructure() throws SynapseException {
+        EntityChildrenRequest request = new EntityChildrenRequest();
+        request.setParentId(project.getId());
+        request.setIncludeTypes(Lists.newArrayList(EntityType.folder));
+        EntityChildrenResponse response = synapse.getEntityChildren(request);
+        assertNotNull(response);
+        assertNotNull(response.getPage());
+        assertEquals(2, response.getPage().size());
```

Comment:
Not clear what `testNavigateFolderStructure` is testing.  There's no reference to the logic in SynapseUtil and this code need not test that the Synapse client works.